dist/install-otp.sh

```diff
@@ -17,4 +17,4 @@ tar zxf "${FILE_OUTPUT}" -C "${DIR_FOR_BIN}" --strip-components=1
 echo "Installed Erlang/OTP version follows"
 ${DIR_FOR_BIN}/bin/erl -version
 
-echo "INSTALL_DIR_FOR_OTP=${RUNNER_TEMP}/${DIR_FOR_BIN}" >> "${GITHUB_ENV}"
```

Comment:
This change was a mistake as it stopped setting the env var which was done via writing to GITHUB_ENV file. Ill try to send a fix tomorrow. Sorry about that.

Reply:
No prob. I should've reviewed better... and also tests should've probably failed (unless that's not important for tests).